app/src/pages/OnDeviceDisplay/ProtocolDetails/index.tsx

```diff
@@ -25,67 +26,80 @@ import {
 } from '@opentrons/react-api-client'
 import { ProtocolResource } from '@opentrons/shared-data'
 import { MAXIMUM_PINNED_PROTOCOLS } from '../../../App/constants'
-import { MediumButton, TabbedButton } from '../../../atoms/buttons'
+import { MediumButton, SmallButton, TabbedButton } from '../../../atoms/buttons'
 import { Chip } from '../../../atoms/Chip'
 import { StyledText } from '../../../atoms/text'
-import { SmallModalChildren } from '../../../molecules/Modal/OnDeviceDisplay'
+import { useMissingHardwareChipText } from '../../../organisms/OnDeviceDisplay/RobotDashboard/hooks'
+import {
+  Modal,
+  SmallModalChildren,
+} from '../../../molecules/Modal/OnDeviceDisplay'
 import { useToaster } from '../../../organisms/ToasterOven'
 import { getPinnedProtocolIds, updateConfigValue } from '../../../redux/config'
+import { useMissingProtocolHardware } from '../../Protocols/hooks'
 import { Deck } from './Deck'
 import { Hardware } from './Hardware'
 import { Labware } from './Labware'
 import { Liquids } from './Liquids'
 
+import type { ModalHeaderBaseProps } from '../../../molecules/Modal/OnDeviceDisplay/types'
 import type { Dispatch } from '../../../redux/types'
 import type { OnDeviceRouteParams } from '../../../App/types'
-import { deleteProtocol, deleteRun, getProtocol } from '@opentrons/api-client'
 
 const ProtocolHeader = (props: {
   title: string
   handleRunProtocol: () => void
+  chipText: string
 }): JSX.Element => {
   const history = useHistory()
   const { t } = useTranslation(['protocol_info, protocol_details', 'shared'])
-  const { title, handleRunProtocol } = props
-
+  const { title, handleRunProtocol, chipText } = props
   const [truncate, setTruncate] = React.useState<boolean>(true)
   const toggleTruncate = (): void => setTruncate(value => !value)
 
   let displayedTitle = title
-
   if (title.length > 92 && truncate) {
     displayedTitle = truncateString(title, 92, 69)
   }
 
-  //  TODO(ew, 3/23/23): put real info in the chip
-
   return (
     <Flex
       alignItems={ALIGN_CENTER}
       justifyContent={JUSTIFY_SPACE_BETWEEN}
-      margin={SPACING.spacing16}
-      marginBottom={SPACING.spacing40}
+      marginX={SPACING.spacing16}
+      paddingY={SPACING.spacing32}
+      position={POSITION_STICKY}
+      top={0}
+      backgroundColor={COLORS.white}
     >
       <Flex
         alignItems={ALIGN_CENTER}
         gridGap={SPACING.spacing16}
         marginBottom={SPACING.spacing8}
+        width="674px"
       >
         <Btn
           paddingLeft="0rem"
-          paddingRight="1.25rem"
+          paddingRight={SPACING.spacing20}
           onClick={() => history.goBack()}
-          width="2.5rem"
+          width={SPACING.spacing40}
```

Comment:
For now, let's avoid using spacing for width and height since when Brian updated SPACING, he used hardcode value for a case like this.

|```suggestion
          width="2.5rem"
```